docs/ref/models/expressions.txt

```diff
@@ -79,6 +79,26 @@ Built-in Expressions
 
 .. class:: F
 
+    .. method:: asc(nulls_first=False, nulls_last=False)
+
+        Returns the expression ready to be sorted in ascending order.
+
+        ``nulls_first`` and ``nulls_last`` define how null values are sorted.
+
+        .. versionchanged:: 1.11
+
+            The ``nulls_last`` and ``nulls_first`` parameters were added.
+
+    .. method:: desc(nulls_first=False, nulls_last=False)
+
+        Returns the expression ready to be sorted in descending order.
+
+        ``nulls_first`` and ``nulls_last`` define how null values are sorted.
+
+        .. versionchanged:: 1.11
+
+            The ``nulls_first`` and ``nulls_last`` parameters were added.
+
```

Comment:
You're right that `asc` and `desc` were not documented for `F`, but maybe it's worth to mention that something was changed.